genie-agent/src/main/java/com/netflix/genie/agent/cli/GlobalAgentArguments.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package com.netflix.genie.agent.cli;
+
+import lombok.Getter;
+
+/**
+ * Global command-line options for the agent.
+ *
+ * @author mprimi
+ * @since 4.0.0
+ */
+@Getter
+class GlobalAgentArguments {
```

Comment:
Is this a holder for something in the future?

Reply:
yes, global options (e.g., log level)